sdk-api.yaml

```diff
@@ -98,25 +119,28 @@ paths:
   /organizations/{organizationIdentifier}/merge:
     put:
       operationId: mergeOrganization
+      summary: Merge organization
+      description: Merges an organization to another, and as a result, migrates all of that organization's users to the new organization.
       tags:
+        - Organizations
         - stage
       parameters:
         - $ref: "#/components/parameters/OrganizationIdentifierParameter"
       security:
         - StageApiKey: []
       requestBody:
         content:
-          application/json:
+          application/vnd.heystage.v1+json:
```

Comment:
Interesting change.

`application/vnd.heystage.v1+json` is probably correct, but what I was thinking was versioning the _response_, not the _request_. I'm not sure if versioning the request makes sense or not.

Any thoughts on this change?

Reply:
What do you think @kaiserstage and @les-stage? It seems like the request should stay `application/json` because we're versioning the response, not the request.

Reply:
Versioning the request should allow us to respect multiple versions - if a client makes a request with an outdated version and we still understand how to ingest it, we can allow the request to succeed.

Reply:
The OpenAPI spec dictates that if `accept` is added as a header parameter, [it should be ignored by tool providers.](https://spec.openapis.org/oas/v3.0.3#fixed-fields-9). Hence I changed all the content-types... I suspect we should be taking the media-type from the response and displaying it in the request. Let me do some research on how we should be handling these situations.

Reply:
Confirmed. Making a new commit with the changes. We'll need to implement this on our side as well, will take a few days.